install_config/cluster_metrics.adoc

```diff
@@ -1031,6 +1035,254 @@ Once `openshift_metrics_project: openshift-infra` is installed, metrics can be
+information.
+
+*Volume Operation Metrics*
+
+These metrics report time taken by a storage operation once started. These
+metrics keep track of operation time at the plugin level, but do not include
+time taken by `goroutine` to run or operation to be picked up from the internal
+queue. These metrics are of type Histogram.
+
+.Example summary of available volume operation metrics
+----
+storage_operation_duration_seconds { volume_plugin = "aws-ebs", operation_name = "volume_attach" }
+storage_operation_duration_seconds { volume_plugin = "aws-ebs", operation_name = "volume_detach" }
+storage_operation_duration_seconds { volume_plugin = "glusterfs", operation_name = "volume_provision" }
+storage_operation_duration_seconds { volume_plugin = "gce-pd", operation_name = "volume_delete" }
+storage_operation_duration_seconds { volume_plugin = "vsphere", operation_name = "volume_mount" }
+storage_operation_duration_seconds { volume_plugin = "iscsi" , operation_name = "volume_unmount" }
+storage_operation_duration_seconds { volume_plugin = "aws-ebs", operation_name = "unmount_device" }
+storage_operation_duration_seconds { volume_plugin = "cinder" , operation_name = "verify_volumes_are_attached" }
+storage_operation_duration_seconds { volume_plugin = "<n/a>" , operation_name = "verify_volumes_are_attached_per_node" }
+----
+
+See
+link:https://github.com/kubernetes/community/blob/master/contributors/design-proposals/storage/volume-metrics.md[Volume
+operation metrics] for more information.
+
+*Volume Stats Metrics*
+
+These metrics typically report usage stats of PVC (such as used space vs available space). The type of metrics emitted is gauge.
+
+.Volume Stats Metrics
+|===
+|Metric|Type|Labels/tags
+
+|volume_stats_capacityBytes
+|Gauge
+|namespace,persistentvolumeclaim,persistentvolume=
+
+|volume_stats_usedBytes
+|Gauge
+|namespace=<persistentvolumeclaim-namespace> 
+persistentvolumeclaim=<persistentvolumeclaim-name> 
+persistentvolume=<persistentvolume-name>
+
+|volume_stats_availableBytes
+|Gauge
+|namespace=<persistentvolumeclaim-namespace> 
+persistentvolumeclaim=<persistentvolumeclaim-name> 
+persistentvolume=
+
+|volume_stats_InodesFree
+|Gauge
+|namespace=<persistentvolumeclaim-namespace> 
+persistentvolumeclaim=<persistentvolumeclaim-name> 
+persistentvolume=<persistentvolume-name>
+
+|volume_stats_Inodes
+|Gauge
+|namespace=<persistentvolumeclaim-namespace> 
+persistentvolumeclaim=<persistentvolumeclaim-name> 
+persistentvolume=<persistentvolume-name>
+
+|volume_stats_InodesUsed
+|Gauge
+|namespace=<persistentvolumeclaim-namespace> 
+persistentvolumeclaim=<persistentvolumeclaim-name> 
+persistentvolume=<persistentvolume-name>
+|===
+
+[[k8s-desired-metrics]]
+==== Desired Metrics
+This section describes the desired metrics to be emitted from Kubernete's storage subsystem. These metrics are not currently emitted by the system, but 
+they can help to identify the health of the storage subsystem and cluster.
+
+.Desired Metrics
```

Comment:
There is no need to add Desired metrics section here. It was present in original Google doc for data collection.